rosdep/python.yaml

```diff
@@ -5940,6 +5940,13 @@ python3-cairo:
     slackpkg:
       packages: [py3cairo]
   ubuntu: [python3-cairo]
+python3-cairosvg:
+  arch: [python-cairosvg]
+  debian: [python3-cairosvg]
+  fedora: [python-cairosvg]
```

Comment:
It looks like the Python 3 package is called `python3-cairosvg` on Fedora. @cottsay am I reading the spec file correctly?

https://src.fedoraproject.org/rpms/python-cairosvg/blob/rawhide/f/python-cairosvg.spec#_35

Reply:
Nailed it!

```suggestion
  fedora: [python3-cairosvg]
```